### ISSUE TITLE
Re-evaluate versioning scheme

### ISSUE THREAD

**Issue by kriegaex** [eclipse-aspectj/aspectj#72]
Do we want to continue with the minor-minor versioning since 1.9.x?

How about making AspectJ follow the Java major in its minor, i.e. the next release could be AspectJ 1.17 (not even 1.17.0, only after that 1.17.1, if a maintenance release is necessary).

If we ever upgrade to 2.x whenever it is justified (not now), we could simply skip 2.0 and continue with e.g. 2.x for Java release x, the major only signifying a major update of AspectJ as such.

**Comment by sbrannen** [comment]
- see also: https://github.com/eclipse-aspectj/aspectj/issues/329#issuecomment-2763332538
